.gitignore

```diff
@@ -67,5 +67,6 @@ __screenshots__/
 
 # build files
 dist/
+!/docs/public/dist/wasm-lz4.wasm # need this symlink to stay there
```

Comment:
Had to add a symlink for `npm run docs` to work properly with static file hosting.